dom/timer/timer-dom.js

```diff
@@ -1,2 +1,44 @@
-console.log("time to start the timer!");
+let time = document.querySelector('p')
+
+let startButton = document.querySelector('button')
+let stopButton = document.querySelector('.btn2')
+let resetButton = document.querySelector('.btn3') 
+
+let timerId;
+let counter = 10;
+
+time.innerHTML = counter
+
+function start() {
+    if (!counter) {
+        reset();
+    } else {
+        clock();
+    }
+}
+  
+function stop() {
+    if (timerId) {
+        clearInterval(timerId);
+        timerId = null;
+    }
+}
+  
+function reset() {
+    stop();
+    counter = 10;
+    time.innerHTML = counter
+}
+  
+function clock() {
+    timerId = setInterval(function() {
```

Comment:
Every time the start button is clicked, it creates another setInterval. It makes the time go supper fast after you click 5 times.